docs/participant_endpoint_enums.md

```diff
@@ -0,0 +1,37 @@
+# Participant Endpoint Enums
+
+This is the list of endpoints that a PISP or DFSP must register with the swith to get callbacks from the `thirdparty-api-adapter` and `transaction-request-service`.
+
+These endpoints should be added to the enums of `central-services-shared`, and the seeds of `central-ledger`
+
+## PISP:
+
+- `THIRDPARTY_CALLBACL_URL_CONSENT_REQUEST_PUT`
+- `THIRDPARTY_CALLBACK_URL_CONSENT_REQUEST_PUT_ERROR`
+- `THIRDPARTY_CALLBACK_URL_CONSENT_POST`
+- `THIRDPARTY_CALLBACK_URL_CONSENT_REQUEST_PUT`
```

Comment:
```suggestion
- `THIRDPARTY_CALLBACK_URL_CONSENT_PUT`
```

Reply:
do we need `THIRDPARTY_CALLBACK_URL_CONSENT_PUT_ERROR` ?

Reply:
Yeah you're probably right.